cardinal/t_gameSystems_test.go

```diff
@@ -624,6 +684,122 @@ func TestGetPlayersPresence_OtherPlayerAround(t *testing.T) {
 
 // #endregion Look System Test
 
+// #region Inventory System Test
+func TestInventory_WithItem(t *testing.T) {
+	tf := testutils.NewTestFixture(t, nil)
+	MustInitWorld(tf.World)
+	setup()
+
+	const playerName = "Hueyu"
+	const playerID = 3
+	const roomSpawn = 0
+
+	tokens := []string{"take", "ball"}
+	var expectedOutInventory string = ("You have a Football.")
+	var expectedOutTake string = ("You picked up a Football.")
+	var expectedOutErr uint8 = 0
+
+	// Create an initial player
+	_ = tf.AddTransaction(getCreateMsgID(t, tf.World), msg.CreatePlayerMsg{
+		PlayersName: playerName,
+		RoomID:      roomSpawn,
+	})
+	tf.DoTick()
+
+	outputTake, er := system.Take(tokens, playerID, roomSpawn, ts, cardinal.NewWorldContext(tf.World))
+
+	tf.DoTick()
+
+	output, er := system.Inventory(playerID, ts, cardinal.NewReadOnlyWorldContext(tf.World))
+
+	assert.Equal(t, expectedOutInventory, output)
+	assert.Equal(t, expectedOutTake, outputTake)
+	assert.Equal(t, expectedOutErr, er)
+}
+
+func TestInventory_WithNOItem(t *testing.T) {
```

Comment:
this test should reallt tests that the player inventory has a ball as well as the output

Reply:
This test is testing if the player has no item in his inventory and that the output which is a string. The test if the player has the ball is  `TestInventory_WithItem`